.gitignore

```diff
@@ -1,3 +1,6 @@
+# Temporary Admin Functionality
+admin*
+
```

Comment:
what's this trying to ignore?

Reply:
It was something I was experimenting with and didn't want to push. Removed now